packages/atomic/src/components/recommendations/atomic-recs-result/atomic-recs-result.tsx

```diff
@@ -161,8 +161,9 @@ export class AtomicRecsResult {
     }
     return (
       // deepcode ignore ReactSetInnerHtml: This is not React code
```

Comment:
I reckon you don't need deepcode ignore comment, just address the notice in GH
(@btaillon amarite?)

Reply:
I didn't know you could address it in GH. Either it wasn't there last time I checked, or I never thought about using this feature. I have a small preference for keeping it in the code though, since that's searchable.

Reply:
I don't know why this particular ignore is on that line though, that looks wrong.